goose3/network.py

```diff
@@ -62,3 +59,9 @@ def fetch(self, url):
                 raise NetworkError(response.status_code, response.reason)
 
         return text
+
+    def fetch_obj(self, url):
+        return self._connection.get(url, timeout=self.config.http_timeout,
```

Comment:
nit: does PEP 8 require timeout to be on the next line?

Reply:
Nope, pep 8 doesn't require them all to be on different lines. We could actually move it to two lines since flake8 is checking for 120 character width.